packages/fluentui/react-northstar/src/themes/teams/components/Dialog/dialogVariables.ts

```diff
@@ -21,15 +22,17 @@ export interface DialogVariables {
   headerFontWeight: number;
 
   headerActionMargin: string;
+  footerActionsBreakpoint: string;
 }
 
 export const dialogVariables = (siteVariables): Partial<DialogVariables> => ({
   border: 'none',
 
   rootBackground: siteVariables.colors.white,
   rootBorderRadius: siteVariables.borderRadiusXLarge,
-  rootWidth: '50vw',
+  rootWidth: '600px',
```

Comment:
is 600px intentional here?

Reply:
yes, it's intentional, because without fixed width (with 50vw) it's impossible to reach max-width 100vw to be able to see dialog full screen on screens smaller than 600px.

And number 600px is based on examples that Josh has provided.